app/services/filter_service.rb

```diff
@@ -0,0 +1,22 @@
+class FilterService
+  def self.filter_by_search(base_collection, search_term = nil)
+    if search_term.present?
+      base_collection.search_by(search_term)
+    else
+      base_collection
+    end
+  end
+
+  def self.filter_case_logs(logs, search_term, filters, all_orgs, user)
```

Comment:
This was copied from `case_logs_filter`, which isn't specced. Not sure if we want to remedy that, or if we're happy that the page tests (which will make use of it) are sufficient.